dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -21,18 +21,18 @@ ENV NJOBS="2" \
     OPAMROOTISOK="true"
 
 # Base opam is the set of base packages required by Coq
-ENV COMPILER="4.02.3"
+ENV COMPILER="4.05.0"
 
 RUN opam init -a -y -j $NJOBS --compiler="$COMPILER" default https://opam.ocaml.org && eval $(opam config env) && opam update
 
 # Common OPAM packages.
 # `num` does not have a version number as the right version to install varies
 # with the compiler version.
-ENV BASE_OPAM="num ocamlfind.1.8.0 dune.1.1.1 ounit.2.0.8" \
+ENV BASE_OPAM="num ocamlfind.1.8.0 dune.1.1.1 ounit.2.0.8 odoc.1.2.0" \
```

Comment:
Why move odoc to BASE_OPAM? Its users haven't changed AFAICS.

Reply:
`odoc` is ocaml >= 4.03 [see https://opam.ocaml.org/packages/odoc/] IMHO it makes sense to have it at the base as we may want to build the API docs in all settings.

Anyways it can be kept like that. Dune release still requires >= 4.06